docs/cloud/dev/spryker-cloud-commerce-os/multi-store-setups/multi-store-setups.md

```diff
@@ -63,4 +63,4 @@ With the separated setup, store share the same codebase but have dedicated datab
 
 ### Separated setup: Integration
 
-New projects are shipped with a separated setup by default. If your project is still on a shared setup, [integrate multi-database logic](/docs/scos/dev/technical-enhancement-integration-guides/integrate-multi-database-logic.html).
+New projects are shipped with a shared setup by default. To switch to a separated setup, reach out to your Sales Representative.
```

Comment:
Shouldn't it be "Sales Representative" => "sales representative"
```suggestion
New projects are shipped with a shared setup by default. To switch to a separated setup, reach out to your sales representative.
```

Reply:
Should Technical Writer be technical writer? :D

Reply:
@andriitserkovnyi As far as I remember, we discussed this question and decided to write job titles in lowercase=)
Though, to my best knowledge, the capitalization of job titles depends on the context; sometimes it's should be capitalized, and sometimes not.